package.json

```diff
@@ -60,7 +60,7 @@
   },
   "dependencies": {
     "@azure/arm-storage": "^10.1.0",
-    "@azure/keyvault-secrets": "^4.0.0-preview.5",
+    "@azure/keyvault-secrets": "4.0.0-preview.5",
```

Comment:
@sarath-p note I've pinned this at `preview.5`. They seem to change their module structure in newer versions and breaks our code.

Reply:
Sounds good. We can unpin when they release a non=preview version.